### ISSUE TITLE
Add option to reverse touchpad scrolling

### ISSUE THREAD

**Issue by Jacalz** [fyne-io/fynedesk#197]
I now have FyneDesk up and running on my Fedora Laptop and I noticed that the touch pad scrolling scrolls the view and not the content. I prefer to have the touch pad scroll the content and not the view, just like how it is on the phones and tablets. I think it would be a great addition in the settings to have this option. It is also very important for me that this does not change the scrolling of the mouse as that still needs to behave as normal :)

**Comment by andydotxyz** [comment]
I wonder if this is a request for Fyne itself? All apps probably want the same outcome in this regard

**Comment by Jacalz** [comment]
I'm not entirely sure if that's needed. I have the option changed in Gnome and all Fyne applications are working just as I would expect. I think it's just changing the scroll input to the applications.

**Comment by mattboll** [comment]
I would love this option too, but unlike Jacalz I want to reverse the mouse scrolling too :p

**Comment by andydotxyz** [comment]
Agreed, I think @Jacalz was responding to my consideration of it being Fyne specific.
Seems like it is indeed a FyneDesk config

**Comment by Jacalz** [comment]
To clarify, I strongly want it to not "mess up" my mouse scrolling, I want it to just change my touchpad scroll direction. :)

**Comment by andydotxyz** [comment]
Oh. So when you run Fyne apps inside FyneDesk then the mouse scrolling is correct but touchpad is not?

**Comment by Jacalz** [comment]
I mean that I want it to behave like reversed for touchpad but not for mouse. It is currently the same direction for mouse and touchpad.

**Comment by andydotxyz** [comment]
OK, thanks for the clarification. I guess two settings may be required then, to satisfy everyone ;)
